### DISCUSSION TITLE
about `lsp_declarations` and `lsp_definitions`

### DISCUSSION THREAD

**Discussion by Masber** [ibhagwan/fzf-lua#1480] [Q&A] [answered]
In telescope, `lsp_declarations` and `lsp_definitions` takes you directly to the line of code directly while fzf-lua opens fzf results related to the lsp query.

I find the telescope functionality more appropriate since both declaration and definition are unique per symbol.

Would it be possible to replicate the same telescope feature in the fzf-lua configuration?

thank you

**Comment by ibhagwan** [comment, marked as answer]
Use `jump_to_single_result=true` the docs are there for a reason, read the Wiki:
https://github.com/ibhagwan/fzf-lua/wiki#lsp-single-result

**Reply by Masber** [reply]
yeah.. sorry I did not see the wiki, I went straight to the plugin docs in neovim... the wiki is pretty good

**Comment by ibhagwan** [comment]
It’s also the default in the “telescope” profile.
